2200008/1.py

```diff
@@ -0,0 +1,15 @@
+# 반반
+import sys
+sys.stdin = open('input.txt')
+
+for t in range(1, int(input())+1):
+    s = input()
+    set_s = set(s)
+    if len(set_s) != 2:
+        print(f'#{t} No')
+        continue
+    for chr in set_s:
+        if s.count(chr) !=2:
+            print(f'#{t} No')
+            break
+    print(f'#{t} Yes')
```

Comment:
현재 코드는 입력값이 `AAAB`와 같이 같은 문자가 3번 등장할 때 한 케이스에 대해 2 줄을 출력합니다!
```suggestion
    for chr in set_s:
        if s.count(chr) !=2:
            print(f'#{t} No')
            break
    else:
        print(f'#{t} Yes')
```
위와 같이 수정해주시면 됩니다 😄